File: cw/python/calculate_average.py
# https://www.codewars.com/kata/57a2013acf1fa5bfc4000921/train/python


def find_average(numbers):
    sum = 0

    for i in numbers:
        sum += i

    return sum / len(numbers)


if __name__ == "__main__":
    lim = int(input())
    numbers = []
    for i in range(lim):
        i = int(input())
        numbers.append(i)

    print(find_average(numbers))

File: codewars/cpp/calculate_average.cpp
// https://www.codewars.com/kata/57a2013acf1fa5bfc4000921


#include <bits/stdc++.h>
using namespace std;

double calcAverage(const vector<int>& values) {
    double sum = 0;
    for (int i = 0; i < values.size(); ++i) {
        sum += values[i];
    }

    return (sum / values.size());
}

int main() {
    int numbers, n;
    vector<int> values;

    cin >> n;

    for(int i = 0; i < n; ++i) {
        cin >> numbers;
        values.push_back(numbers);
    }

    cout << calcAverage(values) << endl;
}